runtime/acala/src/lib.rs

```diff
@@ -2034,10 +2030,10 @@ mod benches {
 	// frame_benchmarking::define_benchmarks!(
 	// 	// XCM
 	// 	[pallet_xcm, PalletXcmExtrinsicsBenchmark::<Runtime>]
+	// // TODO: add oracle
```

Comment:
```suggestion
```

Reply:
the oracle benchmarks are removed in this PR and need to add it back later using the benchmarks in the ORML repo